MTM-49961/MTM-47816 remove esper reference from microservice SDK… (#730)
SoftwareAG/c8y-docs: content/microservice-sdk/rest-bundle/microservice-development.md

}]
```

> After the handshake, the client needs to subscribe to the particular device channel or all devices channel (*). This is done using a POST request with the name of the channel. 

Reviewer: ```suggestion
After the handshake, the client must subscribe to the particular device channel or all devices channel (*). This is done using a POST request with the name of the channel. 
```

Author: done

> Let's assume that in our example we want to receive notifications about events from the device with internal id 24800. In this case the subscription channel is "/events/24800".

Reviewer: ```suggestion
For example, if you want to receive notifications about events from the device with internal ID "24800", the subscription channel is "/events/24800".
```

Author: done
